package.json

```diff
@@ -215,7 +215,6 @@
     "s2-geometry": "^1.2.10",
     "use-local-storage-state": "^9.0.2",
     "uuid": "^3.2.1",
-    "webpack-bundle-analyzer": "^3.6.0",
```

Comment:
removed this since it doesn't play nice with webpack-dev-server with respect to outputting tangible results:

https://github.com/webpack-contrib/webpack-bundle-analyzer/issues/147#issuecomment-380737178